Program.cs

```diff
@@ -233,6 +233,47 @@ static async Task Main(string[] _)
 
             await ReadyEvent.OnStartup(discord);
             
+            // Migration from joinwatch to user notes
```

Comment:
This should either run before connecting the client, or as part of the startup event. If the discord client isn't required then before is better, if it's needed then do it inside the startup event to avoid polluting main.

Also break this out into its own function, maybe even with its own file (could do a new folder called Migrations?)